README.md

```diff
@@ -1,11 +1,10 @@
 # Repository for the TERRA REF Data Paper
 
-
 This is a data descriptor, intended for submission to Nature Scientific Data as citable reference that briefly describes and points to our data products and computing pipeline.
 
 We have an inclusive opt-in authorship policy- if you contributed and your name isn't listed in the [author list](https://docs.google.com/spreadsheets/d/1FnaeJZ1A6r1fa3UvhfczMLKh5CWaSMErBRw4vTKgPe8/edit?usp=sharing) please read the [TERRA REF authorship guidelines](https://terraref.gitbooks.io/terraref-documentation/content/manuscripts-and-authorship.html) and [fill out the authorship form](https://goo.gl/forms/xgzvF2LppMAZrrHx2)
```

Comment:
```suggestion
We have an inclusive opt-in authorship policy- if you contributed and your name isn't listed in the [author list](https://docs.google.com/spreadsheets/d/1FnaeJZ1A6r1fa3UvhfczMLKh5CWaSMErBRw4vTKgPe8/edit?usp=sharing) please read the [TERRA REF authorship guidelines](https://docs.terraref.org/user-manual/manuscripts-and-authorship-guidelines#co-authorship) and [fill out the authorship form](https://goo.gl/forms/xgzvF2LppMAZrrHx2)
```
For some reason, the original link took me to main terra ref docs page, but the link I suggested takes me to the specific authorship page.